lecture14/tests/conftest.py

```diff
@@ -0,0 +1,29 @@
+import configparser
+
+import pytest
+from selenium import webdriver
+
+from lecture14.pageobjects.page_objects import *
+
+
+@pytest.fixture(scope="session")
+def user():
+    config = configparser.ConfigParser()
+    config.read('properties/user.ini')
+    return User(config.get('User', 'username'), config.get('User', 'password'))
+
+
+@pytest.fixture(scope="function")

```

Comment:
There could be session scope, no need to create new webdriver for each test

Reply:
Fixed.

Reply:
One thing: in this case we have the second test depends on the first, I tried to do independently. It is hardly important in homework, but it is easy to implement both options

Reply:
Done.